commons/utils/src/main/java/in/projecteka/jataayu/util/ui/DateTimeUtils.kt

```diff
@@ -40,7 +40,7 @@ class DateTimeUtils {
             val relativeTimeSpan = getDate(createdAt)?.let {
                 DateUtils.getRelativeTimeSpanString(it.time).toString()
             }
-            return relativeTimeSpan ?: "Something went wrong.Please update the consent expiry date and try again"
+            return relativeTimeSpan ?: "unknown"
```

Comment:
This can be declared as a constant or can be moved to a string file

Reply:
sure